CM_ "AUTOGENERATED FILE, DO NOT EDIT"


CM_ "Imported file _comma.dbc starts here"
BO_ 512 GAS_COMMAND: 6 EON
 SG_ GAS_COMMAND : 7|16@0+ (0.253984064,-83.3) [0|1] "" INTERCEPTOR
 SG_ GAS_COMMAND2 : 23|16@0+ (0.126992032,-83.3) [0|1] "" INTERCEPTOR
 SG_ ENABLE : 39|1@0+ (1,0) [0|1] "" INTERCEPTOR
 SG_ COUNT : 45|2@0+ (1,0) [0|3] "" INTERCEPTOR
 SG_ CHECK : 43|4@0+ (1,0) [0|3] "" INTERCEPTOR

BO_ 513 GAS_SENSOR: 6 INTERCEPTOR
 SG_ INTERCEPTOR_GAS : 7|16@0+ (0.253984064,-83.3) [0|1] "" EON
 SG_ INTERCEPTOR_GAS2 : 23|16@0+ (0.126992032,-83.3) [0|1] "" EON
 SG_ STATE : 39|8@0+ (1,0) [0|255] "" EON
 SG_ COUNT : 45|2@0+ (1,0) [0|3] "" EON
 SG_ CHECK : 43|4@0+ (1,0) [0|3] "" EON

VAL_ 513 STATE 5 "FAULT_TIMEOUT" 4 "FAULT_STARTUP" 3 "FAULT_SCE" 2 "FAULT_SEND" 1 "FAULT_BAD_CHECKSUM" 0 "NO_FAULT" ;


CM_ "Imported file _honda_2017.dbc starts here"
VERSION ""


NS_ :
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BU_: PCM MACC ABS EPS EON ADAS XXX

BO_ 228 STEERING_CONTROL: 5 ADAS
 SG_ STEER_TORQUE : 7|16@0- (1,0) [-3840|3840] "" EPS
 SG_ STEER_TORQUE_REQUEST : 23|1@0+ (1,0) [0|1] "" EPS
 SG_ SET_ME_X00 : 22|7@0+ (1,0) [0|127] "" EPS
 SG_ SET_ME_X00_2 : 31|8@0+ (1,0) [0|255] "" EPS

BO_ 198 STEERING_SENSORS: 8 PCM
 SG_ STEER_ANGLE_RATE : 23|16@0+ (0.1,-3276.8) [0|65535] "deg/s" XXX
 SG_ STEER_ANGLE : 7|16@0+ (0.1,-3276.8) [0|65535] "deg" XXX

BO_ 390 POWERTRAIN_DATA: 7 PCM
 SG_ PEDAL_GAS : 47|16@0+ (1,0) [0|65535] "" XXX
 SG_ RPM : 7|16@0+ (0.1,0) [0|65535] "rpm" XXX

BO_ 502 BRAKE: 8 PCM
 SG_ BRAKE_PRESSED : 19|1@0+ (1,0) [0|1] "" XXX

 BO_ 666 FRONT_SPEEDS: 8 PCM
 SG_ WHEEL_SPEED_FL : 23|16@0+ (0.005,0) [0|65535] "kmh" XXX
 SG_ WHEEL_SPEED_FRONT : 39|16@0+ (0.005,0) [0|65535] "kmh" XXX
 SG_ WHEEL_SPEED_FR : 7|16@0+ (0.005,0) [0|65535] "kmh" XXX

BO_ 668 REAR_SPEEDS: 8 PCM
 SG_ WHEEL_SPEED_RL : 7|16@0+ (0.005,0) [0|65535] "kmh" XXX
 SG_ WHEEL_SPEED_RR : 23|16@0+ (0.005,0) [0|65535] "kmh" XXX

BO_ 679 SPEEDS: 8 XXX
 SG_ SPEED1 : 7|16@0+ (0.005,0) [0|65535] "" XXX
 SG_ SPEED2 : 23|16@0+ (0.005,0) [0|65535] "" XXX

BO_ 649 MACCHINA: 8 MACC
 SG_ STEER_TORQUE_DRIVER : 7|16@0- (1,0) [-31000|31000] "" EON
 SG_ MAIN_ON : 16|1@0+ (1,0) [0|1] "" EON
 SG_ ACC_STATUS : 20|1@0+ (1,0) [0|1] "" EON  
 SG_ STEER_STATUS : 35|4@0+ (1,0) [0|15] "" EON
 SG_ STEER_CONTROL_ACTIVE : 36|1@0+ (1,0) [0|1] "" EON
 SG_ CRUISE_SETTING : 41|2@0+ (1,0) [0|3] "" EON
 SG_ CRUISE_BUTTONS : 46|3@0+ (1,0) [0|7] "" EON
 SG_ CRUISE_SPEED_OFFSET : 55|8@0- (0.1,0) [-128|127] "kmh" EON
 SG_ CRUISE_SPEED_PCM : 63|8@0+ (1,0) [0|255] "kmh" EON
 SG_ GEAR_SHIFTER : 24|4@1+ (1,0) [0|15] "" EON
 SG_ GEAR : 31|4@0+ (1,0) [0|15] "" EON 

VAL_ 649 GEAR_SHIFTER 32 "L" 16 "S" 8 "D" 4 "N" 2 "R" 1 "P" ;
VAL_ 649 GEAR 7 "L" 10 "S" 4 "D" 3 "N" 2 "R" 1 "P" ;
VAL_ 649 CRUISE_BUTTONS 7 "tbd" 6 "tbd" 5 "tbd" 4 "accel_res" 3 "decel_set" 2 "cancel" 1 "main" 0 "none" ;
VAL_ 649 CRUISE_SETTING 3 "distance_adj" 2 "tbd" 1 "lkas_button" 0 "none" ;